src/popup.js

```diff
@@ -169,4 +318,7 @@ document
 const isTest = getUrlParameter("test") === "true";
 
 // Display extension status when popup is opened
+// FIXME Since this is async, the popup is supposedly loaded before its status.
+//       Fixing this should make tests simpler and require less waiting for
+//       element visibility.
```

Comment:
After this comment was made, you and I discussed this in person. I've removed the `FIXME` message because the supposed fault only affects tests and we can work around it very easily.